spectec/spec/9-module.watsup

```diff
@@ -147,15 +147,15 @@ def $rundata(DATA byte* (ACTIVE x instr*), y) =
 
 def $instantiate(store, module, externval*) : config
 def $instantiate(s, module, externval*) = s'; f; instr_e* instr_d* (CALL x)?
-  -- if module = MODULE type* import* func^n_func global* table* mem* elem* data* start? export*
+  -- if module = MODULE (TYPE rectype)* import* func^n_func global* table* mem* elem* data* start? export*
```

Comment:
Can we move matching the TYPE header into the alloctypes functions? That would be more consistent.

Reply:
Makes sense, that would be much better.